pkgs/games/factorio/default.nix

```diff
@@ -17,7 +17,7 @@ let
   variants = {
     x64 = {
       url = "https://www.factorio.com/get-download/${version}/alpha/linux64";
-      sha256 = "1xpzrx3q678519qgjl92fxn3qv55hd188x9jp6dcfk2ljhi1gmqk";
+      sha256 = "1dc53swnlbz23czw8v3ldl4ayp7118qyja4vh1z5dsd871swz9qg";
     };
 
     x32 = {
```

Comment:
One way is to download the tarball somehow and run `nix-hash --flat --type sha256 --base32 archive.tar` or use `nix-prefetch-url`.